examples/examples_test.py

```diff
@@ -90,7 +90,7 @@ def testKernelRegressionTrainAndPredict(self):
     ys = jnp.dot(xs, truth)
     kernel = lambda x, y: jnp.dot(x, y)
```

Comment:
You might try using highest precision for the matmuls. On TPU, this may be important.

Reply:
Nice, Precision.HIGH does the trick! (On TPU, I'm assuming this won't hurt GPU/CPU)